Dfe.Academies.Academisation.Domain/ApplicationAggregate/Application.cs

```diff
@@ -398,5 +407,21 @@ public CommandResult DeleteSchool(int urn)
 
 		return new CommandSuccessResult();
 	}
+
+	public CommandResult Delete(int applicationId)
+	{
+		var validationResult = new DeleteApplicationValidator().Validate(applicationId);
+
+		if (!validationResult.IsValid)
+		{
+			return new CommandValidationErrorResult(
+				validationResult.Errors.Select(x => new ValidationError(x.PropertyName, x.ErrorMessage)));
+		}
+
+
+		return new CommandSuccessResult();
```

Comment:
There is no deletion happening here, should we be returning a CommandSuccessResult? Might just be an oddity with the pattern but this is just Validation passing not a 'Delete'